Firestore/core/src/firebase/firestore/nanopb/byte_string.h

```diff
@@ -44,6 +44,10 @@ class ByteString : public util::Comparable<ByteString> {
  public:
   ByteString() = default;
 
+  /**
```

Comment:
One final question (unrelated to this line): We've got pretty strong ownership semantics around `pb_bytes_array_t* bytes_`. Would it make sense to formalize it a bit further by converting it to `std::unique_ptr<pb_bytes_array_t> bytes_`?